mne/viz/utils.py

```diff
@@ -1950,7 +1950,7 @@ def _check_cov(noise_cov, info):
 
 
 def _set_title_multiple_electrodes(title, combine, ch_names, max_chans=6,
-                                   all=False, ch_type=None):
+                                   all_=False, ch_type=None):
```

Comment:
I rename the argument because of the function `all()`, and made sure it wasn't used as a kwargs across `mne`. 
LMK if you want me to revert if you think other packages in the ecosystem use this function and might use `all=` in the function call.

Reply:
if they do, their CIs will hopefully tell us :)